src/components/buttonComponent/buttonComponent.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import './buttonComponent.css';
+
+class ButtonComponent extends Component {
+
+  render() {
+    return (
+      <div className={this.props.divClassName}>
+          <a className={this.props.linkClassName} href={this.props.url}>
```

Comment:
Out of curiosity, what caused you to change this to use `href`?

Reply:
We wanted to use the button component to substitute the buttons on the start page, but we couldn't figure out how to use onClick to route to the pages we wanted (we have to pass in a function to onClick for the restart button). So we decided to just use href instead to stream line all the buttons instead of creating separate button components for the start page and restart button.